.travis.yml

```diff
@@ -1,21 +1,30 @@
 language: python
 python:
   - "3.6"
-  - "3.7-dev" 
+  - "3.7-dev"
 
 # command to install dependencies
 install:
   - pip install -r requirements.txt
+  - pip install coveralls
 
 before_script:
   - psql -c "CREATE DATABASE testauthor;" -U postgres
   - psql -c "ALTER user postgres PASSWORD 'pass';" -U postgres
   - export DATABASE_URL="postgresql://postgres:pass@localhost/testauthor"
+  - export SENDGRID_KEY="sengridkey"
```

Comment:
@martinMutuma can we kindly have the `SENDGRID_KEY` and the actual api key  added in the Travis settings `environment` section.

Reply:
@Tevinthuku I don't think it's needed for tests because no actual email is sent when running tests in Django. I just have those as place holders to prevent them from raising an error. I posted the actual variables which include my account name and password in the group

Reply:
So the tests don't send an actual email to the user?